source/shared/cpp/ObjectModel/Enums.cpp

```diff
@@ -128,6 +130,7 @@ namespace AdaptiveSharedNamespace
             {AdaptiveCardSchemaKey::TextWeight, "weight"},
             {AdaptiveCardSchemaKey::Thick, "thick"},
             {AdaptiveCardSchemaKey::Thickness, "thickness"},
+            {AdaptiveCardSchemaKey::Thickness, "thickness"},
```

Comment:
Duplicate

Reply:
Also, while you're in the neighborhood, looks like AdaptiveCardSchemaKey::Thick is not in use anywhere.

---
In reply to: [253137890](https://github.com/Microsoft/AdaptiveCards/pull/2351#discussion_r253137890) [](ancestors = 253137890)